### ISSUE TITLE
Convert manual deployment to Auto deployment as per commit message.

### ISSUE THREAD

**Issue by WasiqB** [WasiqB/maven-deploy#5]
#### Is your feature request related to a problem? Please describe.
Currently when any of deployment job is triggered (e.g.: patch release pipeline is triggered, then only one job is completed. Other jobs remain on hold thus holding up the complete workflow. This causes Slack message to be skipped and no message is received for the build status.

#### Describe the solution you'd like
Deployment should be automated and only one job should be used. It will determine the type of release depending on PR merge to master message, it will check the start of the message and it should start with `[release:major|minor|patch|skip]`
